### DISCUSSION TITLE
[Feature]: Support for Samswung SRW raw format

### DISCUSSION THREAD

**Discussion by AGJSpiegelenberg** [immich-app/immich#1599] [Feature Request]
### Feature detail

Currently SRW raw files are not being uploaded to the server. Will be great if this format can be supported.

### Platform

Server

**Comment by samip5** [comment]
Example file please so I can test what it thinks about it?

**Comment by AGJSpiegelenberg** [comment]
No problem.

I had to split the zip file for it being to big for github.
[Samsung - RAW Format.zip.001.zip](https://github.com/immich-app/immich/files/10487363/Samsung.-.RAW.Format.zip.001.zip)
[Samsung - RAW Format.zip.002.zip](https://github.com/immich-app/immich/files/10487364/Samsung.-.RAW.Format.zip.002.zip)

**Comment by samip5** [comment]
> No problem.
> 
> I had to split the zip file for it being to big for github. [Samsung - RAW Format.zip.001.zip](https://github.com/immich-app/immich/files/10487363/Samsung.-.RAW.Format.zip.001.zip) [Samsung - RAW Format.zip.002.zip](https://github.com/immich-app/immich/files/10487364/Samsung.-.RAW.Format.zip.002.zip)

Just to verify, the picture is just red?

<img src="https://files.skym.fi/p8ST5BwQk6NJHMZ/direct">

**Comment by AGJSpiegelenberg** [comment]
That is correct, today was the red office day.

**Comment by uhthomas** [comment]
Fixed by #2834.

**Comment by ChrisS85** [comment]
Not sure if relevant here, but in my case the previews are displaying but the images are not loading when I click on them, only the jpeg files from the camera are displayed. I'm getting some kind of timeout and then an "error loading image" with a placeholder picture.
